src/viam/sdk/services/motion/server.cpp

```diff
@@ -178,6 +178,96 @@ ::grpc::Status MotionServer::GetPose(::grpc::ServerContext* context,
     return ::grpc::Status();
 };
 
+::grpc::Status MotionServer::GetPlan(::grpc::ServerContext* context,
+                                     const ::viam::service::motion::v1::GetPlanRequest* request,
```

Comment:
I was expecting the request here to be a const reference like the grpc stub API. Once again, this is clearly the existing pattern, but curious if you're aware of any motivation here or if it's just incidental.

Reply:
This is the API for the proto `MotionService::Service` that we're inheriting from so I think we're committed to it. I think this is a difference between the [server](https://github.com/viamrobotics/viam-cpp-sdk/blob/e1b1e4801e2d916918cf40e414ec32b6e743a450/src/viam/api/service/motion/v1/motion.grpc.pb.h#L333) and [client](https://github.com/viamrobotics/viam-cpp-sdk/blob/e1b1e4801e2d916918cf40e414ec32b6e743a450/src/viam/api/service/motion/v1/motion.grpc.pb.h#L105) code.